index.html

```diff
@@ -157,87 +157,87 @@ <h2>Stages</h2>
 
 <h2>Input into the process</h2>
 
-<p>Ideas for evolving the ECMAScript language are accepted in any form. Any discussion, idea, or proposal for a change or addition which has not been submitted as a formal proposal is considered to be a “strawperson” (stage 0) and has no acceptance requirements. Such submissions must either come from TC39 delegates or from non-delegates who have <a href="https://tc39.github.io/agreements/contributor">registered</a> via Ecma International.
+<p>Ideas for evolving the ECMAScript language are accepted in any form. Any discussion, idea, or proposal for a change or addition which has not been submitted as a formal proposal is considered to be a strawperson (Stage 0) and has no acceptance criteria. Such submissions must either come from TC39 delegates or from non-delegates who have <a href="https://tc39.github.io/agreements/contributor">registered</a> via Ecma International.
 
 <h2>Spec revisions and scheduling</h2>
 
-<p>TC39 intends to submit a specification to the ECMA General Assembly for ratification in July of each year. The following is an approximate timeline for producing a new spec revision:</p>
+<p>TC39 intends to submit a specification to the Ecma General Assembly for ratification in July of each year. The following is an approximate timeline for producing a new spec revision:</p>
 <ul>
   <li>February 1: Candidate Draft is produced.</li>
   <li>February - March: 60 day royalty-free opt-out period.</li>
-  <li>March TC39 Meeting: stage 4 proposals are incorporated, final semantics are approved, and the new spec version is branched from <code>main</code>. Only editorial changes are accepted from this point forward.</li>
-  <li>April-June: ECMA CC and ECMA GA review period.</li>
-  <li>July: Approval of new standard by the ECMA General Assembly</li>
+  <li>March TC39 Meeting: Stage 4 proposals are incorporated, final semantics are approved, and the new spec version is branched from <code>main</code>. Only editorial changes are accepted from this point forward.</li>
+  <li>April-June: Ecma ExeCom and Ecma GA review period.</li>
+  <li>July: Approval of new standard by the Ecma General Assembly</li>
```

Comment:
Abbreviations for Ecma sub-units should be expanded and linked just like "_Ecma <a href="https://ecma-international.org/technical-committees/tc39/">TC39</a> (Technical Committee 39)_" in the first paragraph.
```suggestion
  <li>April-June: Ecma [ExeCom](https://ecma-international.org/organisation/#executive-committee) (Executive Commitee) and [GA](https://ecma-international.org/organisation/#general-assembly) (General Assembly) review period.</li>
  <li>July: Approval of new standard by the Ecma GA</li>
```

Reply:
agree with the change, but the full name should come first, e.g. Ecma Executive Committee (ExeCom).  I will make this change